mautic/mautic
# New manually unsubscribed from email or SMS filters

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | Y
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Required: )
#### Description:
It is requested to enable users to filter for *manually added DNC contacts* for **email** and **sms**.
Fields labeled:
 * Manually marked as do not contact - Email
 * Manually marked as do not contact - SMS

#### Steps to test this PR:
1. Create few segments containing current DNC reasons and their combination
2. Write down the numbers so you can verify
3. Checkout this PR and CC
4 . Verify that filters you have created in **1** are still valid.
5. Create segment containing new DNC fields and verify the content is valid.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10972"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
